empress/vistrans/ReconRenderer.py

```diff
@@ -0,0 +1,418 @@
+import RenderClasses
+import tkinter as tk
+
+
+NODE_RADIUS = 5
+TREE_OFFSET = [20, 30]
```

Comment:
Could you explain in a comment what the two numbers mean? What is the tree offset, and why are there two values?